packages/playground-examples/copy/en/4-0/New TS Features/Unknown in Catch.ts

```diff
@@ -0,0 +1,36 @@
+//// { compiler: { ts: "4.0.1" } }
+
+// Because JavaScript allows throwing any value, TypeScript
+// does not support declaring the type of an error
+
+try {
+  // ..
+} catch (e) {}
+
+// Historically, this has meant that the `e` in the catch
+// would default to any. This allowed for the freedom to
+// arbitrarily access any property. With 4.0, we've loosened
+// the restrictions on type assignment in the catch clause
+// to allow both `any` and `unknown`.
+
+// Same behavior with any:
+try {
+  // ..
+} catch (e) {
+  e.stack;
+}
+
+// Explicit behavior with unknown:
+
+try {
+  // ..
+} catch (e) {
```

Comment:
does this actually show off the feature? I thought it required a type annotation.

Reply:
err, hrm, I swear I added the `:any` and `:unknown` - good spot

Reply:
Yep, prettier wipes them!

Reply:
Reported, https://github.com/prettier/prettier/issues/8646